crates/services/txpool_v2/src/tests/universe.rs

```diff
@@ -293,6 +295,12 @@ impl TestPoolUniverse {
         }
     }
 
+    pub fn assert_pool_integrity(&self, expected_txs: &[ArcPoolTx]) {
+        let pool = self.pool.as_ref().unwrap();
+        let pool = pool.read();
+        pool.assert_integrity(expected_txs);
```

Comment:
It is the place were we can call `assert_integrity` for all internal fields of the `TxPool`.

Reply:
Done.